scripts/shared-program.js

```diff
@@ -1,18 +1,11 @@
-import { graphqlQueryNameList } from "./graphql.js";
-import { getProductIconMapping, getBaseConfigPath } from './site-config.js';
+//import { executeQuery, graphqlQueryNameList } from "./graphql.js";
```

Comment:
@mdickson-adbe can you remove this commented code

Reply:
Done!